test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -0,0 +1,393 @@
+
+        //try to hit an admin restricted page and see that we are unauthorized
+        given().cookie("JSESSIONID", cookie).when().get("https://localhost:9993/admin/index.html").then().log().all().assertThat().statusCode(403);
+    }
+
+    @Test
+    public void testBasicRestAccess() throws Exception {
+        String url = SERVICE_ROOT + "/catalog/query?q=*";
+
+        configureRestForBasic();
+
+        //test that we get a 401 if no credentials are specified
+        when().get(url).then().log().all().assertThat().statusCode(401);
+
+        //try a random user and get a 401
+        given().auth().basic("bad", "user").when().get(url).then().log().all().assertThat().statusCode(401);
+
+        //try a real user and get an sso token back
+        String cookie = given().auth().basic("admin", "admin").when().get(url).then().log().all()
+                .assertThat().statusCode(200)
+                .assertThat().header("Set-Cookie", containsString("JSESSIONID")).extract().cookie("JSESSIONID");
+
+        //try the sso token instead of basic auth
+        given().cookie("JSESSIONID", cookie).when().get(url).then().log().all().assertThat().statusCode(200);
+
+        //try that admin level sso token on a restricted resource and get in... sso works!
+        given().cookie("JSESSIONID", cookie).when().get("https://localhost:9993/admin/index.html").then().log().all().assertThat().statusCode(200);
+    }
+
+    private void configureRestForAnonymous() throws IOException, InterruptedException {
+        PolicyProperties policyProperties = new PolicyProperties();
+        policyProperties.put("authenticationTypes", "/=SAML|ANON,/admin=SAML|basic,/jolokia=SAML|basic,/system=SAML|basic,/solr=SAML|PKI|basic");
+        policyProperties.put("whiteListContexts", "/services/SecurityTokenService,/services/internal,/proxy," + SERVICE_ROOT + "/sdk/SoapService");
+        Configuration config = configAdmin.getConfiguration(PolicyProperties.FACTORY_PID, null);
+        Dictionary<String, ?> configProps = new Hashtable<>(policyProperties);
+        config.update(configProps);
+        waitForAllBundles();
+    }
+
+    private void configureRestForBasic() throws IOException, InterruptedException {
+        PolicyProperties policyProperties = new PolicyProperties();
+        policyProperties.put("authenticationTypes", "/=SAML|basic,/admin=SAML|basic,/jolokia=SAML|basic,/system=SAML|basic,/solr=SAML|PKI|basic");
+        policyProperties.put("whiteListContexts", "/services/SecurityTokenService,/services/internal,/proxy," + SERVICE_ROOT + "/sdk/SoapService");
+        Configuration config = configAdmin.getConfiguration(PolicyProperties.FACTORY_PID, null);
+        Dictionary<String, ?> configProps = new Hashtable<>(policyProperties);
+        config.update(configProps);
+        waitForAllBundles();
+    }
+
+    @Test
+    public void testAnonymousSoapAccess() throws Exception {
+        configureRestForAnonymous();
```

Comment:
It is specific to REST, but I am also adding a white list for my soap service to the policy. Since I don't know which test is actually going to run first, I include it here to ensure that the policy is correct before starting up.